luaui/configs/keyboard_layouts.lua

```diff
@@ -76,7 +76,47 @@ scanToCode["dvorak"] = {
 	-- NEEDS CORRECTION ABOVE
 }
 
-local layouts = { 'qwerty', 'qwertz', 'azerty', 'dvorak' }
+scanToCode["de-neo"] = {
+	Q = "X",
+	W = "V",
+	E = "L",
+	R = "C",
+	T = "W",
+	Y = "K",
+	U = "H",
+	I = "G",
+	O = "F",
+	P = "Q",
+	A = "U",
+	S = "I",
+	D = "A",
+	F = "E",
+	G = "O",
+	H = "S",
+	J = "N",
+	K = "R",
+	L = "T",
+	Z = "Ü",
+	X = "Ö",
+	C = "Ä",
+	V = "P",
+	B = "Z",
+	N = "B",
+	M = "M",
+	[";"] = "d",
+	["'"] = "y",
+	[","] = ",",
+	["."] = ".",
+	["/"] = "j",
+	["`"] = "^",
+	["-"] = "-",
+	["="] = "`",
+	-- NEEDS CORRECTION BELOW
+	["\\"] = "\\",
```

Comment:
If this is correct you don't need the comment above and below

Reply:
Thanks for the review!

It's not correct and it's rather tricky to correct this. The key which is on qwerty used for the `\` is used as modifier for the third level on NEO. So it's not even a dead key like the accents, but just triggers a change for all other keys.

I'm not sure how to properly implement this in the keymap.

Reply:
I see, thank you for clarification.

Reply:
Keep in mind these layouts are just a workaround for engine not having translation layer from scancode -> key label yet (used only for presentation purposes)

So it's not a big deal.